config/custom-environment-variables.json

```diff
@@ -1,3 +1,4 @@
 {
-  "jwtPrivateKey" : "jwt_secretKey"
+  "jwtPrivateKey" : "jwt_secretKey",
+  "db": "db_url"
 }
```

Comment:
always leave a newline at the end of your files

Reply:
Okay.